app/src/Submit/OnlineJudgeFactory/Szkopul.ts

```diff
@@ -0,0 +1,49 @@
+/*
+    cpbooster "Competitive Programming Booster"
+    Copyright (C) 2022  ???
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+import { Page } from "playwright-chromium";
+import OnlineJudge, { OnlineJudgeName } from "./OnlineJudge";
+
+export default class CS3233 extends OnlineJudge {
```

Comment:
Lets use the name of the Online Judge as the name of the class. Something like
```suggestion
export default class Szkopul extends OnlineJudge {
```

Reply:
Oh sorry, sent the wrong patch. (why does it even compile?)

Reply:
It compiles because it is a default export, you can name it however you want where you import the class.